src/lib/components/Buttons/Logout/index.svelte

```diff
@@ -6,14 +6,25 @@
   import { supabase } from '$lib/utils/functions/supabase';
   import { capturePosthogEvent } from '$lib/utils/services/posthog';
 
-  async function logout() {
-    const { error } = await supabase.auth.signOut();
-    console.error('Error logging out: ', error);
 
-    capturePosthogEvent('user_logged_out');
-    posthog.reset();
-    goto('/login');
+async function handleLogout(){
+  document.cookie = 'access_token=;expires=' + new Date().toUTCString() + ';path=/;';
+  document.cookie = 'refresh_token=;expires=' + new Date().toUTCString() + ';path=/;';
+}
+  async function logout() {
+    //const { error } = await supabase.auth.signOut();
+    //console.error('Error logging out: ', error);
+ try {
+  await handleLogout();
+  window.location.href= '/login';
+  capturePosthogEvent('user_logged_out');
+  posthog.reset();
+ }catch(error){
+  console.error('Logout error', error)
+ }
+    
```

Comment:
@cannarron Thanks for the PR.

A couple of things:
1. We don't set cookies manually within the application, so there is no need for the `handleLogout` function

2. All you just need to do is upon logout you reset the value of the organization name. The first thing to have done was look at what was responsible for showing the organisation name on the signup or login pages and then reset that property.

    The way to implement this would be to get the store that holds the organisation info and then clear the org data.
    
    ```js
    // `currentOrg` is the store with the organisation info
    // `orgs` is the list of organisation the loggedin user belongs to
    import { currentOrg, orgs } from '$lib/utils/store/org';
    
    // then in the logout function you set the values to empty strings
    
    async function logout() {
      const { error } = await supabase.auth.signOut();
      console.error('Error logging out: ', error);
    
      // Reset current org info
      $currentOrg = {
        id: '',
        name: '',
        shortName: '',
        siteName: '',
        avatar_url: '',
        memberId: '',
        role_id: '',
        landingpage: {},
        theme: ''
      }
      // Reset to org list
      $orgs = []
    
      capturePosthogEvent('user_logged_out');
      posthog.reset();
      goto('/login');
    }
    ```
3. Please make sure you test that it works and add a video showing that the bug ticket is fixed

Reply:
Okay

Reply:
Thank you for taking the time to fix this issue.😊 

Please don't mark an issue as resolved, only the creator of the issue should do that.

If you think about it, if I reported an issue, I should be the one to also mark it as resolved.

And in addition, I don't see any new commit which means the comment hasn't been addressed.

Look forward to your updates 😉😃